libmscore/fret.cpp

```diff
@@ -1071,6 +1091,32 @@ FretItem::Barre FretDiagram::barre(int f) const
       return FretItem::Barre(-1, -1);
       }
 
+//---------------------------------------------------------
+//   setHarmony
+///   if this is being done by the user, use undoSetHarmony instead
+//---------------------------------------------------------
+
+void FretDiagram::setHarmony(QString harmonyText)
```

Comment:
I guess this is needed to support having chord symbol in Inspector, and I'm not opposed to that.  But I'd also exp[ect it to work to simply add the chord symbol normally.  That is, when adding a chord symbol to a segment that has a fret diagram, we should add it to the fret digram, not the segment.  And certainly, if user selects the fret diagram and pressed Ctrl+K to enter a chord symbol, it should be added there.  That's probably the most substantive issue I have with any of this.  It's solvable, though.

Reply:
Yes, I agree. I'll see what I can do to implement it.